sdk/avalon_sdk/ethereum/proxy/ethereum_worker_registry.py

```diff
@@ -0,0 +1,243 @@
+                    "failed", "Invalid application id {}".format(org_id))
+            lookupResult = self.__contract_instance.functions.workerLookUpNext(
+                worker_type.value,
+                org_id, application_id, lookup_tag).call()
+            return lookupResult
+        else:
+            logging.error(
+                "worker registry contract instance is not initialized")
+            return construct_message("failed", "worker registry contract \
+                instance is not initialized")
+
+    def worker_register(self, worker_id, worker_type, organization_id,
+                        application_type_ids, details, id=None):
+        """
+        Registering a new Worker
+        """
+        if (self.__contract_instance is not None):
+            if (is_valid_hex_str(binascii.hexlify(worker_id).decode("utf8"))
+                    is False):
+                logging.error("Invalid worker id {}".format(worker_id))
+                return None
+            if not isinstance(worker_type, WorkerType):
+                logging.error("Invalid workerType {}".format(worker_type))
+                return None
+            if is_valid_hex_str(binascii.hexlify(organization_id)
+                                .decode("utf8"))is False:
+                logging.error("Invalid organization id {}"
+                              .format(orgnization_id))
+                return None
+            if not _is_valid_json(details):
+                logging.error("Invalid details {}".format(details))
+                return None
+            for app_id in app_type_ids:
+                if (is_valid_hex_str(binascii.hexlify(app_id).decode("utf8"))
+                        is False):
+                    logging.error("Invalid application id {}".format(app_id))
+                    return None
+
+            txn_hash = self.__contract_instance.functions.workerRegister(
+                worker_type, organization_id, application_type_ids, details)
+            .buildTransaction(self.__eth_client
+                              .get_transaction_params())
+            txn_status = self.__eth_client.execute_transaction(
+                txn_hash)
+            return txn_status
+        else:
+            logging.error(
+                "worker registry contract instance is not initialized")
+            return None
+
+    def worker_update(self, worker_id, details, id=None):
+        """
+        Updating a Worker
+        """
+        if (self.__contract_instance is not None):
+            if (is_valid_hex_str(binascii.hexlify(worker_id).decode("utf8"))
+                    is False):
+                logging.error("Invalid worker id {}".format(worker_id))
+                return None
+            if not _is_valid_json(details):
+                logging.error("Invalid details {}".format(details))
+                return None
+
+            txn_status_hash = self.__contract_instance.functions.workerUpdate(
```

Comment:
Any specific reason why this variable is names as txn_status_hash where as other functions name this variable as txn_hash

Reply:
No specific reason Manoj. Updated all the txn variables to be more appropriate.